Update README.md

lilyjma opened the pull request:
to match with the link name under Examples section

richardpark-msft commented:
I have this open PR (with this also incorporated) if you want to take a look. There were a few other issues as well:
https://github.com/Azure/azure-sdk-for-js/pull/8256

I preserved the "from" and just adjusted the target instead. Do you prefer 'using'?

lilyjma commented:
> I have this open PR (with this also incorporated) if you want to take a look. There were a few other issues as well:
> #8256
> 
> I preserved the "from" and just adjusted the target instead. Do you prefer 'using'?

Nope. I chose 'using' just to line up with the line above.

HarshaNalluru commented on sdk/servicebus/service-bus/README.md:
````diff
@@ -186,7 +186,7 @@ await sender.send({
 ```
 
-### Receive messages from Sessions
+### Receive messages using Sessions
````
`from` is more appropriate here.
`Receive messages from session-enabled-entities` is more accurate.

ramya-rao-a commented:
Can we have this PR closed since the consensus seems to be towards using "from"?
